### DISCUSSION TITLE
Where is the local repo the plugin commit to before pushing?

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#143] [Q&A — Plugin] [answered]
I think the plugin should have a local repo to before pushing, right? Where is it? I guess it says somewhere in [here](https://github.com/ObsidianPublisher/obsidian-github-publisher/tree/master/plugin/publish) but can't figure out. And is there a way to have it also push changes in the assets folder?

**Comment by Mara-Li** [comment, marked as answer]
There is no local repo. 
And no, as assets don't have frontmatter/metadata.

**Reply by ooker777** [reply]
so what git commands does it do?

**Reply by Mara-Li** [reply]
The plugin push directly on the remote repo using the API.
